packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -129,6 +170,12 @@ To enable grouped batching:
     }
 ```
 
+## Note about performance and latency
+
+In terms of performance and impact on latency, the results greatly depend on payload size, payload structure, network speed and CPU speed. Therefore, customers must perform the required measurements and adjust the settings according to their scenarios.
+
+In general, compression offers a trade-off between higher compute costs, lower bandwidth consumption and lower storage requirements, while chunking slightly increases latency due to the overhead of splitting an op, sending the chunks and reconstructing them on each client. Grouped batching heavily decreases the number of ops observed by the server and slightly decreases the bandwidth requirements as it merges all the ops in a batch into a single op and also eliminates the op envelope overhead.
+
 ## How it works
```

Comment:
I think this sections belongs to line 68 where you were talking about Compression.